src/include/daos_srv/daos_server.h

```diff
@@ -192,6 +192,7 @@ struct dss_xstream;
 
 bool dss_xstream_exiting(struct dss_xstream *dxs);
 bool dss_xstream_is_busy(void);
+daos_epoch_t dss_get_start_epoch(void);
```

Comment:
Nit-picking: "epoch" is typically only used in TX and I/O contexts; "time", "timestamp", etc. should be more consistent.

Reply:
Currently, the start epoch is only used for comparing with the RPC epoch, from such perspective, the name "start epoch" may be more clear than "start timestamp", consider the use case of "if (rpc_epoch < start_timestamp)".

Reply:
We can revisit the naming later.